azurelinuxagent/common/protocol/wire.py

```diff
@@ -642,6 +657,23 @@ def stream(self, uri, destination, headers=None, use_proxy=None):
 
         return success
 
+    def fetch_content_and_etag(self, uri, headers=None, use_proxy=None, decode=True):
+        logger.verbose("Fetch [{0}] with headers [{1}]", uri, headers)
+        content = None
+        etag = None
+        not_modified = False
```

Comment:
NIT: Same comment as below, could you please avoid using double negatives. 
Eg:  Change `not_modified=False` to `modified=True`

Reply:
see comment above